src/admin/__init__.py

```diff
@@ -166,6 +166,29 @@ def manage_user(email):
     )
 
 
+@admin.route("/manage_orders", methods=["GET"])

```

Comment:
These two could be wrapped in a blueprint, so that this route becomes
```python
manage_orders_blueprint = Blueprint("manage_orders", __name__, url_prefix="/manage_orders", static_folder="../static")
admin.register_blueprint(manage_orders_blueprint)


@manage_orders_blueprint("",methods=["GET"])
...
...
```
Which I think is cleaner since it makes it harder to make typos when writing new code.